src/views/wallet/Transfer.vue

```diff
@@ -121,7 +121,7 @@
                                 block
                                 :disabled="!canSendAgain"
                             >
-                                Start Again
+                                New Transaction
```

Comment:
If it's not translated and you change such place - please add translation

Reply:
no need for translation, this pr contains changes for camino-suite and the translation is not implemented yet